### PR TITLE
Image carousel front end design and functionality

### PR DESCRIPTION
**Pull request by cstung21**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import $ from 'jquery';
+import styled from 'styled-components';
+import Carousel from './Carousel.jsx';
+
+const exampleData = require('../../../exampleData.js');
+
```
Comment (brandonjunus) [inline client/src/components/App.jsx:7]: Example data will eventually be replaced with real data from the ajax call, correct?

**Review by brandonjunus — APPROVED** [review]
I would refactor the left and right arrows into normal styled components just to simplify maintenance. Other than that, looks good!

File: client/src/components/App.jsx (hunk 1 above)
Comment (cstung21) [reply client/src/components/App.jsx:7]: correct!

**Comment by cstung21** [comment]
Thanks for the suggestion. I've refactored my code to remove the unnecessary arrow components and have instead converted them into styled components

**Pull request merged**